protected/humhub/modules/activity/widgets/views/activityStream.php

```diff
@@ -16,11 +16,13 @@
 
 ?>
 
-
-<div class="panel panel-default panel-activities">
+<div class="panel panel-default panel-activities" id="panel-activities">
+   <?= \humhub\widgets\PanelMenu::widget(['id' => 'panel-activities']); ?>
     <div class="panel-heading"><?= Yii::t('ActivityModule.widgets_views_activityStream', '<strong>Latest</strong> activities'); ?></div>
+   <div class="panel-body">
     <?= Html::beginTag('div', $options) ?>
         <ul id="activityContents" class="media-list activities" data-stream-content>
         </ul>
```

Comment:
Could you place </ul> on line 24?

Reply:
I can do this, the reason I didn't was that Atom auto line breaks to clean my code up like I like it. lol